RedHat-UX/red-hat-design-system
# docs: padding issues

Fixes: https://github.com/RedHat-UX/red-hat-design-system/issues/1080

## What I did

- [ ] Ensure there is 5.0rem (80px) of padding in between content and the Related elements or patterns and Feedback sections

![Screenshot 2023-06-30 at 11 10 24 AM](https://github.com/RedHat-UX/red-hat-design-system/assets/3428964/1c4e02ce-4782-41ed-b6a5-f26f03bc90de)




- [ ] Ensure the padding in between an image container and a table is 2.0rem (32px) (Example: On the Style page under the Light and dark themes h3)

I added a more generic selector that applies the 32px margin between all items except for p, .feedback, copy-permalink which have their own custom margin.

![Screenshot 2023-07-05 at 11 17 57 AM](https://github.com/RedHat-UX/red-hat-design-system/assets/3428964/a67814b5-a3f6-4abf-bd3a-c06e84218611)



- [ ] Ensure the padding in between an h2 and h3 is 2.0rem (32px)

![Screenshot 2023-07-05 at 12 13 52 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/3428964/05627ac5-5b3a-4115-a187-f5c73249288f)



## Testing Instructions

1.  Go to http://localhost:8080/elements/call-to-action/guidelines/
2. Ensure the following
- [ ] there is 80px of padding in between content and the Related elements or patterns and Feedback sections
- [x] padding in between an h2 and h3 is 32px (between "Writing Content" and "Call to action text labels")
- [x] padding in between an image container and a table is 32px (Example: On the Style page under the Light and dark themes h3)
